alchemy-web/src/pages/privacy.astro

```diff
@@ -0,0 +1,113 @@
+---
+/**
+ * Alchemy Privacy Policy
+ * Generated automatically from repository inspection of telemetry implementation.
+ * This page describes how the Alchemy CLI and related tools collect and use data.
+ */
+export const prerender = true; // generate at build time
+
+/**
+ * If your site has a global layout you can wrap this page.
+ * We keep this file standalone to avoid coupling to layouts that may not exist.
+ */
+---
+
+<style>
+  :root {
+    --content-max-width: 60ch;
+  }
+  main {
+    max-width: var(--content-max-width);
+    margin-inline: auto;
+    padding: 2rem 1rem;
+    line-height: 1.6;
+  }
+  h1, h2, h3 {
+    line-height: 1.25;
+    margin-top: 2rem;
+  }
+  h1 {
+    font-size: 2rem;
+    margin-top: 0;
+  }
+  ul {
+    padding-left: 1.25rem;
+  }
+</style>
+
+<main>
+  <h1>Privacy&nbsp;Policy</h1>
+  <p>Last updated: <!-- Will be substituted at build time -->{new Date().toLocaleDateString()}</p>
+
+  <h2>1.&nbsp;Introduction</h2>
+  <p>
+    Thank you for using <strong>Alchemy</strong>. We respect your privacy and are committed to being transparent
+    about the data we collect, why we collect it, and how you can control it. This Privacy Policy explains how the Alchemy CLI,
+    runtime libraries, and associated websites (collectively, "Alchemy") collect and process information
+    when you install, run, or otherwise use Alchemy.
+  </p>
+
+  <h2>2.&nbsp;Information We Collect</h2>
+  <p>Alchemy is designed to collect <em>anonymous</em> usage analytics that help us understand how our tools are used and improve their reliability and performance. We do <strong>not</strong> collect names, email addresses, source code, or any directly identifying personal information. Specifically, the telemetry system built into the Alchemy CLI gathers the following data:</p>
+
+  <h3>2.1.&nbsp;Identifiers</h3>
+  <ul>
+    <li><strong>User&nbsp;ID&nbsp;(UUID)</strong>&nbsp;- generated once per machine and stored locally under your XDG&nbsp;config directory (e.g. <code>~/Library/Preferences/alchemy</code>). <em>Not collected when running in Continuous Integration (CI) environments.</em></li>
```

Comment:
Windows? Or are we fine with people assuming "some windows directory" because the actual directory here doesn't matter?